src/main/java/jenkins/plugins/openstack/compute/slaveopts/BootSource.java

```diff
@@ -354,6 +354,21 @@ public String toString() {
                 return "Volume from Image";
             }
         }
+
```

Comment:
These methods should be inserted before the VFIDesc class, after the toString() method - match the method order used by the other classes here.